src/media/css/lightbox.styl

```diff
@@ -1,7 +1,7 @@
 @import 'lib';
 
 #lightbox {
-    background: rgba(0,0,0,.75);
+    background: rgba($greyscale-light-grey,.9);
```

Comment:
phil commented that .8 is too transparent last time, agree with keeping things consistent though